src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -0,0 +1,80 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-replicasetdetail-sidebar {
+  background-color: #FAFAFA;
+  min-width: 310px;
+}
+
+.kd-replicasetdetail-table td,
```

Comment:
Do not style th or td. Just add a class to specific elements. With that you'll be able to remove !important from the next rule.